core/src/io/anuke/mindustry/graphics/Bloom.java

```diff
@@ -331,7 +331,7 @@ private void gaussianBlur(){
      * @param intensity multiplier for blurred texture in combining phase. must be
      * positive.
      */
-    public void setBloomIntesity(float intensity){
+    public static void setBloomIntesity(float intensity){
```

Comment:
This is a very bad idea. **Never** make things static unless they need to be, especially in a case like this where the bloom can be uninitialized.